scss/application.scss

```diff
@@ -2,3 +2,26 @@
 @import 'variables/_colors';
 @import 'variables/_fonts';
 @import 'variables/_sizes';
+
+.navbar {
+  position: fixed;
+  width: 100%;
+  height: $nav-height;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  -webkit-box-shadow: 0 8px 6px 0 $grey;
+  -moz-box-shadow: 0 8px 6px 0 $grey;
+  box-shadow: 0 8px 6px 0 $grey;
+  padding: 0 15px;
+}
+
+h1 {
+  font: $font-big $font;
```

Comment:
This would be:
`font-family: $font;
font-size: $font-big;`

As the design has all the texts with the same font, you could in the `equalizer.scss` add `font-family: $font` in the selector `*`. Then I would assign it to all the texts.

I forgot something:
The Roboto font needs to be imported in `_font.scss` like this:
@import url ('https://fonts.googleapis.com/css?family=Roboto:400,700');
And after: 
$font: 'Roboto', sans-serif;